apps/frontend/src/app/5_pages/MarketMakingPage/components/MarketMakingTable/MarketMakingTable.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useCallback } from 'react';
+
+import { t } from 'i18next';
+
+import { Table } from '@sovryn/ui';
+
+import { AssetPairRenderer } from '../../../../2_molecules/AssetPairRenderer/AssetPairRenderer';
+import { AssetPairSize } from '../../../../2_molecules/AssetPairRenderer/AssetPairRenderer.types';
+import { translations } from '../../../../../locales/i18n';
+import { AmmLiquidityPool } from '../../utils/AmmLiquidityPool';
+import { AmmLiquidityPoolDictionary } from '../../utils/AmmLiquidityPoolDictionary';
+import { COLUMNS_CONFIG } from './MarketMakingTable.constants';
+import styles from './MarketMakingTable.module.css';
+
+const ammPools = AmmLiquidityPoolDictionary.list();
+
+export const MarketMakingTable: FC = () => {
```

Comment:
```suggestion
export const PoolsTable: FC = () => {
```
Rename the folder + all the related files (types, styles, constants, ...).